src/mergeQueue.ts

```diff
@@ -5,7 +5,8 @@ import {
   removeLabel,
   updatePr,
 } from "./github.ts";
-export const run = async () => {
+import { debounce } from "https://deno.land/std@0.184.0/async/mod.ts";
+const updateBranch = async () => {
```

Comment:
```suggestion

const updateBranch = async () => {
```

Space after import statements.